server/config.js

```diff
@@ -266,9 +266,9 @@ var conf = convict({
     env: 'SESSION_ALLOW_INSECURE_COOKIES'
   },
   server_session_duration: {
-    doc: 'Session duration in milliseconds. Zero, the default, means session cookie.',
+    doc: 'Session duration in milliseconds. Zero means session cookie. We default to one year.',
     format: 'int',
-    default: 0,
+    default: 31556952000,
```

Comment:
LGTM, unless you want to use a constant here, a la
```js
var SEVEN_DAYS_IN_MSEC = 1000 * 60 * 60 * 24 * 7; // 7 days
```

But the `doc` is already pretty darned descriptive.